projects/effortless.md

```diff
@@ -0,0 +1,47 @@
+# Effortless
+
+The Effortless project includes the Habitat Scaffolding for Chef Infra and Chef InSpec and the things to make using them easy.
```

Comment:
This is not the words written in the project.

Please replace this with:

`[Effortless](https://github.com/chef/effortless/blob/master/README.md) is automated best practices for Chef Infra and Chef InSpec.`

Reply:
The Project is more than just what is contained in the Effortless that is why it is different than what is in the Effortless repo.